aws/session/shared_config.go

```diff
@@ -186,6 +197,20 @@ type sharedConfigFile struct {
 	IniData  ini.Sections
 }
 
+// SSOSession provides the shared configuration parameters of the sso-session
+// section.
+type SSOSession struct {
```

Comment:
**Question**: Should this be public?

Reply:
I checked go v2's usage of SSOSession, it is used within config package only so I think we can change it to private